Code/GraphMol/MonomerMol/Conversions.h

```diff
@@ -0,0 +1,38 @@
+/* -------------------------------------------------------------------------
+ * Declares RDKit atomistic ROMol <-> MonomerMol conversion
+ *
+ * Copyright Schrodinger LLC, All Rights Reserved.
+ --------------------------------------------------------------------------- */
+#pragma once
+
+#include <RDGeneral/export.h>
+
+#include <boost/shared_ptr.hpp>
+
+namespace RDKit
+{
+class ROMol;
+class RWMol;
+
+/**
+ * Converts an atomistic ROMol into a Monomer-based ROMol that can
+ * be written to HELM format using the HELM writer.
+ *
+ * @param atomistic_mol Atomistic molecule to convert to MonomerMol
+ * @param use_residue_info Whether to use PDBAtomResidueInfo to determine
+ * monomer boundaries
+ * @return MonomerMol
+ */
+RDKIT_MONOMERMOL_EXPORT boost::shared_ptr<RDKit::RWMol>
```

Comment:
Can't we return `std::unique_ptr` from here? Or, at the least, `std::shared_ptr`?